lib/themes/dosomething/paraneue_dosomething/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "start": "npm run clean && npm run modernizr && webpack",
+    "start": "check-dependencies & npm run clean && npm run modernizr && webpack",
```

Comment:
I could see a good argument for switching it to `&&` to be stricter, but it's also nice to be able to just continue along and build if it's just a patch version difference for example...